ios-base.xcodeproj/xcshareddata/xcschemes/ios-base-develop.xcscheme

```diff
@@ -1,46 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
    LastUpgradeVersion = "1000"
-   version = "1.7">
+   version = "1.3">
```

Comment:
What happened here? 🤔

Reply:
Xcode stuff, maybe the pre-build actions required a higher versions, and deleting those put the version back to 1.3.